File: src/main/java/org/adempiere/exceptions/BPartnerNoBillToAddressException.java
package org.adempiere.exceptions;

import org.compiere.model.I_C_BPartner;

/**
 * Thrown when Bill To Address is required for a BPartner but not found.
 * @author Teo Sarca, www.arhipac.ro
 */
public class BPartnerNoBillToAddressException extends BPartnerException
{
	/**
	 * 
	 */
	private static final long serialVersionUID = -8423260338845096466L;
	public static final String AD_Message = "BPartnerNoBillToAddress";
	
	public BPartnerNoBillToAddressException(I_C_BPartner bp)
	{
		super(AD_Message, bp);
	}
}

File: src/main/kotlin/org/adempiere/exceptions/BPartnerNoBillToAddressException.kt
package org.adempiere.exceptions

import org.compiere.model.I_C_BPartner

/**
 * Thrown when Bill To Address is required for a BPartner but not found.
 * @author Teo Sarca, www.arhipac.ro
 */
class BPartnerNoBillToAddressException(bp: I_C_BPartner) : BPartnerException(AD_Message, bp) {
    companion object {
        /**
         *
         */
        private val serialVersionUID = -8423260338845096466L
        val AD_Message = "BPartnerNoBillToAddress"
    }
}